examples/specs/interactive_multi_line_tooltip.vl.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "Credits to @jakevdp. The example is forked/modified from his block (https://bl.ocks.org/jakevdp/a414950f61e4b224765f2439dd1f09b9).",
+  "data": {
+    "url": "data/stocks.csv",
+    "format": {
+      "type": "csv"
+    }
+  },
+  "width": 800,
+  "height": 400,
+  "layer": [
+    {
+      "mark": "line",
+      "encoding": {
+        "color": {
+          "type": "nominal",
+          "field": "symbol"
+        },
+        "x": {
+          "type": "temporal",
+          "field": "date"
+        },
+        "y": {
+          "type": "quantitative",
+          "field": "price"
+        }
+      }
+    },
+    {
+      "mark": "point",
+      "encoding": {
+        "color": {
+          "type": "nominal",
+          "field": "symbol"
+        },
+        "opacity": {
+          "value": 0,
+          "condition": {
+            "selection": "tooltip",
+            "value": 1
+          }
+        },
+        "x": {
+          "type": "temporal",
+          "field": "date"
+        },
+        "y": {
+          "type": "quantitative",
+          "field": "price"
+        }
+      },
+      "selection": {
+        "tooltip": {
+          "type": "single",
+          "nearest": true,
+          "on": "mouseover",
+          "encodings": [
+            "x"
+          ],
+          "empty": "none",
+          "resolve": "global"
```

Comment:
I just tried myself. empty is definitely critical for initial stage.

However, I still don't understand why you need resolve: "global" since it's actually the default -- see https://vega.github.io/vega-lite/docs/selection.html#resolve

Reply:
Yeah! It's "global" by default. Removing this line.

Reply:
forget to remove this line?

Reply:
My bad. Everything is now updated. You can have a look. :)